nvtabular/loader/tensorflow.py

```diff
@@ -140,20 +140,20 @@ class KerasSequenceLoader(tf.keras.utils.Sequence, DataLoader):
 
     Iterator output is of the form `(dict(features), list(labels))`,
     where each element of the features dict is a
-    `feature_name: feature_tensor`  and each elemtn of the labels
+    `feature_name: feature_tensor`  and each element of the labels
     list is a tensor, and all tensors are of shape `(batch_size, 1)`.
     Note that this means vectorized continuous and multi-hot categorical
     features are not currently supported.
     The underlying NVTabular `Dataset` object is stored in the `data`
     attribute, and should be used for updating NVTabular `Workflow`
-    statistics::
+    statistics:
```

Comment:
Having two :: is actually the correct syntax and not a typo (tells docs to render the next section as a code snippet).

Reply:
This is very good to know, thank you.